addons/mail/models/mail_thread.py

```diff
@@ -1279,6 +1279,10 @@ def _message_parse_extract_payload(self, message, save_original=False):
                     part.set_charset('utf-8')
                 encoding = part.get_content_charset()  # None if attachment
 
+                # Correcting MIME type for PDF files
+                if part.get('Content-Type', '').startswith('pdf;'):
```

Comment:
Shouldn't we use "get_content_type" ?

Reply:
`get_content_type` [is already containing MIME type inferring](https://github.com/python/cpython/blob/54fcfbde300e9c909b56cc67ceac4700ff28ecee/Lib/email/message.py#L617), it will see unrecognized MIME type and give us `text/plain' so we would never know that pdf was there in the first place